src/components/calcite-popover/readme.md

```diff
@@ -3,22 +3,40 @@
 <!-- Auto Generated Below -->
 
 
+## Usage
+
+### AddClickHandle
+
+```html
+<calcite-popover reference-element="popover-button" add-click-handle
```

Comment:
Is there a condition where we don't want a popover to open when the reference element is clicked? For my use cases the overwhelming majority will require this prop, but not sure what other teams needs are?

Reply:
Yeah, in the MapViewer there are some use cases where the reference element is only used for positioning. For example: popover is displayed alongside a panel.